openslides_backend/action/action_handler.py

```diff
@@ -184,6 +184,8 @@ def perform_action(
 
         try:
             write_request, results = action.perform(action_data, self.user_id)
+            if isinstance(write_request, WriteRequest):
```

Comment:
This check seems unnecessary, it is always a WriteRequest; `if write_request` should suffice

Reply:
On another note, we could move this on level up if we want: Currently, if we have an action which executes another action, the required fields are checked one time in the inner action and then another time in the outer action. If we move the check to the `action_handler.py`, we save ourselves the redundancy.

Reply:
The more I think about it the more I like it. Please tell me if you have any arguments against this.

Reply:
Ok, my bad - this is already the upper level.